packages/handlersjs-core/lib/handlers/typed-pipe-through.handler.ts

```diff
@@ -19,22 +19,20 @@ export class TypedPipeThroughHandler<A, B, C, D, E> extends Handler<A, E> {
 
     super();
 
-    if (!this.handlers) {
-
-      throw new HandlerArgumentError('Argument this.handlers should be set.', this.handlers);
-
-    }
+    if (!handlers) { throw new HandlerArgumentError('Argument handlers should be set.', handlers); }
```

Comment:
Should I move these back to the handle() instead of constructor as well?

Reply:
No, this can stay, it's a good catch 👍